posts/2020/07/Three-months-and-having-fun.md

```diff
@@ -0,0 +1,76 @@
+took on a full-time position working on NumPy, funded through a grant to
+[BIDS](https://bids.berkeley.edu). Since April 2020, I have moved to Quansight
+Labs as a full-time developer. 
+
+[Quansight Labs](https://www.quansight.com/labs) is a subsidiary of Quansight
+LLC, and Lab's mission is "Sustaining the future of open source". It does this
+by gathering together an amazing group of software developers and letting them
+loose on a variety of open source projects: Numpy, Spyder, and
+[more](http://labs.quansight.org/categories). Of course, the developers do not
+only volunteer their time. Quansight Labs is sponsored via
+
+- generous support from Quansight LLC,
+- grants like the one from [CZI for NumPy and
+  OpenBLAS](http://labs.quansight.org/blog/2019/11/numpy-openblas-CZI-grant),
+- and via [work
+  orders](http://labs.quansight.org/blog/2019/05/community-driven-opensource-funded-development/)
+  from companies like D. E. Shaw who sponsor [work on
+  Dask](http://labs.quansight.org/blog/2019/08/labs-dask-update).
+
+I personally split my sponsored time between PyTorch and NumPy, and in my free
+time contribute to other projects like PyPy, helping with Python packaging, and
+more.
+
+In the past three months I have met a whole new group of developers. We all
+work remotely, and meet up on Slack and video calls. In the past, the entire
+team would meet once a year or so, until face-to-face gatherings were put on hold. But that does
+not stop the interactions. The organization, with over 20 developers, has
+little hierarchy. Interactions are direct and it is customary to meet up
+for virtual coffee or just to chat. Since we all come from different places
+and backgrounds, there are specialists with deep knowlege in many fields:
+mathematics, C++ and GPU programming, Javascript and web technologies, UX,
+testing, and more that I have not yet explored. The synergy makes it a win-win
+for all involved. When I get stuck, a world-class expert is available and we
+can all help each other move forward.
+
+Lucky me, you say, but why am I bothering to share my good fortune and make
+you all jealous? Here is one take-away for the experienced developers out
+there, keep posted for more as my journey progresses.
+
+I was very hesitant to make the leap into a new career. I am no spring chicken,
+and I was worried that I would not be able to find a suitable position. I only
+began my software developer carreer late in life after I tired of
+electrical design and contracting. But I am here to tell you that even an old
+dog can learn new tricks. I began my open source chapter as a volunteer,
+which improved my programming powers and led to meeting people outside
+my usual circle. This led to the NumPy grant that led to Quansight Labs. So if
+I could do it, others can too.
+
+Joining Quansight Labs was definitely the right step for me. I hope you also
+find your paths enjoyable and rewarding.
+
+Matti Picus
```

Comment:
Since the `:author:` field gets rendered at the top of the post, this is probably not needed? The sentence above is a nice ending.